internal/ui/sponsor_routing_test.go

```
@@ -0,0 +1,93 @@
+
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/x/ansi"
+	"github.com/gfazioli/octoscope/internal/github"
+)
+
+// newSponsorModel builds a Model with the splash open (show_sponsor on,
+// not public-only). GITHUB_TOKEN keeps auth.Token() subprocess-free.
+func newSponsorModel(t *testing.T) Model {
+	t.Helper()
+	t.Setenv("GITHUB_TOKEN", "test-token-not-used")
+	applyTheme("octoscope", "")
```

golangci-lint (2.12.2)

[error] Error return value is not checked

(errcheck)